assets/src/edit-story/components/library/panes/media/media3p/categoryPill.js

```diff
@@ -51,19 +56,28 @@ PillContainer.propTypes = {
   isSelected: PropTypes.bool,
 };
 
-const CategoryPill = (props) => (
-  <PillContainer
-    isSelected={props.isSelected}
-    onClick={props.onClick}
-    role="tab"
-    aria-selected={props.isSelected}
-    data-testid="mediaCategory"
-  >
-    {props.title}
-  </PillContainer>
-);
+const CategoryPill = (props) => {
+  const ref = useRef();
+
+  useRovingTabIndex({ ref, isRowBasedGallery: true });
+
+  return (
+    <PillContainer
+      ref={ref}
+      tabIndex={props.index === 0 ? 0 : -1}
```

Comment:
What is this required for?

Reply:
This is required so the user can tab to the first category initially. After that, they can move from category to category using arrow keys.

I've added a comment, and added isSelected so that the selected category pill can also be tabbed to.